### PR TITLE
Add 8BitDo PC Engine Pad to hid

### PR DESCRIPTION
**Pull request by fmaida**

### CODE REVIEW

**Comment by offalynne** [comment]
@fmaida hi, would you please consider authoring a map for the SDL Community gamepad data repo ? we're missing data on Mac https://github.com/gabomdq/SDL_GameControllerDB

thanks :-)

**Review by RobLoach — CHANGES_REQUESTED** [review]
Thanks, mind adding some additional fields to this, as well as labels and documenation? Some good example:
https://github.com/libretro/retroarch-joypad-autoconfig/blob/master/hid/8BitDo_N30_Pro2_USB.cfg

**Comment by fmaida** [comment]
> Thanks, mind adding some additional fields to this, as well as labels and documenation? Some good example: https://github.com/libretro/retroarch-joypad-autoconfig/blob/master/hid/8BitDo_N30_Pro2_USB.cfg

Whoops, sorry. I'll do that in the next few days.

**Comment by offalynne** [comment]
@RobLoach 

```
input_up_btn_label = "D-Pad Up"
input_down_btn_label = "D-Pad Down"
input_left_btn_label = "D-Pad Left"
input_right_btn_label = "D-Pad Right"
input_select_btn_label = "Select"
input_start_btn_label = "Run"
input_a_btn_label = "I"
input_b_btn_label = "II"
```

should do it.

@fmaida if you could grab me the GUID for SDL with the following tool I would appreciate it: https://www.generalarcade.com/gamepadtool/

**Comment by fmaida** [comment]
I'm not sure if I did it properly, anyway:

```030000000d0f00003801000008010000```

```030000000d0f00003801000008010000,PCEngine PAD,platform:Mac OS X,a:b1,b:b2,back:b8,start:b9,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,```

**Comment by offalynne** [comment]
Thank you so much!

File: hid/PCEngine PAD.cfg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+input_b_btn = "1"
+input_select_btn = "8"
+input_start_btn = "9"
+input_up_btn = "h0up"
+input_down_btn = "h0down"
+input_left_btn = "h0left"
+input_right_btn = "h0right"
+input_a_btn = "2"
+input_gun_trigger_mbtn = "1"
+input_gun_offscreen_shot_mbtn = "2"
```
Comment (RobLoach) [inline hid/PCEngine PAD.cfg:14]:
``` suggestion
input_up_btn_label = "D-Pad Up"
input_down_btn_label = "D-Pad Down"
input_left_btn_label = "D-Pad Left"
input_right_btn_label = "D-Pad Right"
input_select_btn_label = "Select"
input_start_btn_label = "Run"
input_a_btn_label = "I"
input_b_btn_label = "II"
```

**Pull request merged**